travis_build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euv
+
+DEPLOY_VERSION=""
+DEPLOY_BRANCH=""
+if [ "$TRAVIS_PULL_REQUEST" == "true" ]; then
+    echo "Running Production Build"
+    for v in $DEPLOY_PYTHONS; do
+        if [ "$TRAVIS_PYTHON_VERSION" == "$v" ]; then
+            DEPLOY_VERSION=1
+            break
+        fi
+    done
+
+    for b in $DEPLOY_BRANCHES; do
+        if [ "$TRAVIS_BRANCH" == "$b" ]; then
+            DEPLOY_BRANCH=1
+            break
+        fi
+    done
+
+    if [ "$DEPLOY_BRANCH$DEPLOY_VERSION" == "11" ]; then
+        echo "This build will be deployed!"
+        PYB_ARGS="$PYB_ARGS upload"
+    fi
+else:
+    echo "Running PR Build"
+fi
+
+echo ./build.py $PYB_ARGS
```

Comment:
echo?

Reply:
Again, this was all testing of the production script logic in PR.